selenium_tests/ab_tool/ab_tests.py

```diff
@@ -0,0 +1,60 @@
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.support.ui import Select, WebDriverWait
+from selenium.common.exceptions import NoSuchElementException, TimeoutException, NoAlertPresentException
+import unittest, time, re
+from selenium_tests.ab_tool.ab_base_test_case import ABBaseTestCase
+from selenium_tests.ab_tool.page_objects.ab_mainpage import MainPage, Locator1
+from selenium_tests.ab_tool.page_objects.ab_add_experiment_page import AddExperimentPage, Locator2
+
+# These are simple tests to validate that the tool has loaded,
+# as well as validate that an experiment can be added and deleted"""
+
+class test_ab_tool(ABBaseTestCase):
+
+    def test_is_loaded (self):
+        """Check that page is loaded by checking against site title of "A/B Testing Dashboard"""
+        driver = self.driver
+        page = MainPage(driver) #instantiate
+        element = page.get_title()
+        abtext = "A/B Testing Dashboard"
+        print element.text
+        print "Working on verifying that the right page loaded..."
+        driver.save_screenshot('page_after_login.png')
+        self.assertEqual(element.text, abtext, element.text)
```

Comment:
Fixed. Modified with cleaner message, "Error: Wrong page. Expected page title is '{}' but page title is returning '{}'".format(abtext, element.text))